homeassistant/components/rflink.py

```diff
@@ -336,6 +338,17 @@ def _async_handle_command(self, command, *args):
             # if the state is true, it gets set as false
             self._state = self._state in [STATE_UNKNOWN, False]
 
+        # Cover options for RTS
```

Comment:
Not just RTS, I suppose?

Reply:
fixed , also title fixed from Somfy rts to cover, tnx for the heads-up!